.github/workflows/ci.yml

```diff
@@ -28,7 +28,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: errata-ai/vale-action@reviewdog
         with:
-          files: '["docs/overview","docs/subnets", "README.md", "style-checker-notes.md", "style-guide.md"]'
+          files: '["docs/build", "docs/community", "docs/overview","docs/subnets", "README.md", "style-checker-notes.md", "style-guide.md"]'
```

Comment:
I don't see a path called `docs/build` on https://github.com/ava-labs/avalanche-docs/tree/master/docs. Please remove it.

Reply:
There are no errors in the docs/build directory. 

```zsh
vale docs/build
✔ 0 errors, 0 warnings and 0 suggestions in 0 files.
```

Would you like like me to create a new PR w/ just that single addition to the vale files?